micro-rdk/Cargo.toml

```diff
@@ -15,7 +15,7 @@ links = "micro_rdk"
 crate-type = ["lib"]
 
 [features]
-default = ["builtin-components", "data"]
+default = ["builtin-components", "data", "local-signaling"]
```

Comment:
Added a default-enabled feature gate for `local-signaling` so it is easier to disable if we need.